packages/netstring/src/reader.js

```diff
@@ -0,0 +1,71 @@
+// @ts-check
+
+const COLON = ':'.charCodeAt(0);
+
+const decoder = new TextDecoder();
+
+/**
+ * @param {Iterable<Uint8Array> | AsyncIterable<Uint8Array>} input
```

Comment:
This type is different from the one in agoric-sdk; is there a test for the enhancement?

Reply:
This was necessary to satisfy typescript and I believe is related to downleveling iteration, but I’ve really no idea.